package.json

```diff
@@ -36,10 +36,10 @@
     "cross-env": "^5.2.0",
     "fs-extra": "^6.0.1",
     "nodemon": "^1.19.0",
+    "prettier-standard": "^16.1.0",
```

Comment:
Change it to a "fixed" version. The reason of that is that we don't want to break tests if a developer and Jenkins run different versions of the linter. (use `npm install --save-dev --save-exact`)

Reply:
Sounds like a good idea.